shell/platform/darwin/ios/framework/Source/FlutterEngine_Internal.h

```diff
@@ -49,6 +51,16 @@
 - (flutter::PlatformViewIOS*)iosPlatformView;
 
 - (void)waitForFirstFrame:(NSTimeInterval)timeout callback:(void (^)(BOOL didTimeout))callback;
+
+/**
+ * Creates one FlutterEngine from another, sharing components between them.
+ * This results in a faster creation time and a smaller memory footprint engine.
+ *
```

Comment:
If there are expectations, state them here too (such as you need to run before spawning)

Reply:
Addressed in xiao feedback 4 -- added preconditions